parts/k8s/addons/cilium.yaml

```diff
@@ -1,36 +1,55 @@
+---
 apiVersion: v1
-kind: Namespace
+kind: ConfigMap
 metadata:
-  labels:
```

Comment:
@denniszielke We need all these addonmanager labels in order for kube-addon-manager to do anything with this spec. Additionally, we need the Namespace resource because we are defining a new "cilium" resource.

Reply:
yes, my bad - I did not merge ;)
fixed it